spacy/scorer.py

```diff
@@ -1289,3 +1346,22 @@ def lea(input_clusters, output_clusters, mention_to_gold):
         den += len(c)
 
     return num, den
+
+
+# This is coref related, but not from coval.
+# def doc2clusters(doc: Doc, prefix) -> MentionClusters:
```

Comment:
It would be nice to have MentionClusters here but I don't know where to stick it to avoid circular imports.